### DISCUSSION TITLE
Help with installation - Docker or Mac OS

### DISCUSSION THREAD

**Discussion by mkbadri** [chaskiq/chaskiq#477] [Q&A] [answered]
Hi there,

Thanks a lot for initiating this project, looks really promising and I am excited to try it out. I have been trying to implement it locally to test out the solution but even after repeated attempts I end up with the same issue. 

What I tried 
1. docker for development (https://dev.chaskiq.io/en/articles/docker-for-development & https://github.com/chaskiq/chaskiq/issues/414) 
2. Installation on Mac (https://dev.chaskiq.io/en/articles/installation-on-mac)

In both the scenarios I end up with errors on the page when all the components are running - missing translations
![image](https://user-images.githubusercontent.com/10755143/131250466-0efd4c48-a815-4d49-9d5f-4fca9156d8fb.png)

It could be something simple that I am missing or a config issue, unfortunately, I have very limited experience with Ruby and clueless on what that missing element could be. At this point, I am just trying as is system up and for the environment file - it's just the .env.example renamed as .env for now.

Keen to get some help on this and quickly try out the solution locally.

**Comment by michelson** [comment, marked as answer]
https://github.com/chaskiq/chaskiq/issues/482

**Reply by mkbadri** [reply]
Fantastic. that works perfectly now without yarn dev!
